### ISSUE TITLE
Screen size not returned correctly

### ISSUE THREAD

**Issue by wolfshirts** [JoelOtter/termloop#61]
Maybe just an issue with my code, but I seem to be getting 0 back for game.Screen().Size() both width, and height.

**Comment by JoelOtter** [comment]
Please post a minimal executable code sample which reproduces this bug :)

**Comment by wolfshirts** [comment]
Absolutely. Thanks!
```
import (
	"fmt"

	tl "github.com/JoelOtter/termloop"
)

func main() {
	game := tl.NewGame()
	game.Screen().SetFps(30)
	x, y := game.Screen().Size()
	textToDisplay := fmt.Sprintf("Current size x: %d y: %d", x, y)
	text := tl.NewText(10, 10, textToDisplay, tl.ColorBlack, tl.ColorBlue)
	game.Screen().AddEntity(text)
	game.Start()
}
```
<img width="437" alt="Screen Shot 2021-11-23 at 8 08 58 PM" src="https://user-images.githubusercontent.com/43915195/143173358-37acc63a-270f-4210-a46b-7f927063c6d8.png">

**Comment by aquilax** [comment]
Looks like the same issue as https://github.com/JoelOtter/termloop/issues/13

TLDR: Screen size is not initialized before `game.Start()` here https://github.com/JoelOtter/termloop/blob/master/game.go#L89

**Comment by JoelOtter** [comment]
Yep that'll do it :) Until the game has actually started Termloop doesn't know anything about your terminal. Should probably document that but I am still vaguely intending to do a full rewrite of this library given it's _very_ old at this point...
